src/backend/opengl/OpenGLBackend.cpp

```diff
@@ -43,6 +45,52 @@ namespace opengl {
         *device = reinterpret_cast<nxtDevice>(new Device);
     }
 
+    GLuint OpenGLCompareFunction(nxt::CompareFunction compareFunction) {
```

Comment:
nit: both of these could go in the DepthStencil file and could even be made static (or in an anonymous namespace, which is the C++ equivalent)